src/plays/markdown-editor/Readme.md

```diff
@@ -0,0 +1,15 @@
+# Markdown Editor
+
```

Comment:
Please add more details like,

- What someone will learn from this project?
- You have included an NPM, please mention that in the readme.